cfgov/v1/wagtail_hooks.py

```diff
@@ -9,6 +9,7 @@
 @hooks.register('after_create_page')
 @hooks.register('after_edit_page')
 def share_the_page(request, page):
```

Comment:
the page object returned here is actually a `Page` object and not any of the inherited classes that we expected `CFGOV`, `LandingPage` etc. (http://docs.wagtail.io/en/v1.2/reference/hooks.html)